src/API/Rest/InferenceRequest.cs

```diff
@@ -247,8 +258,37 @@ public bool IsValid(out string details)
                 errors.Add($"'inputMetadata' does not yet support type '{InputMetadata?.Details?.Type}'.");
             }
 
+            foreach (var input in InputResources)
+            {
+                if (input.Interface == InputInterfaceType.DicomWeb)
+                {
+                    CheckDicomWebConnectionDetails("inputResources", errors, input.ConnectionDetails);
+                }
+            }
+
+            foreach (var output in OutputResources)
+            {
+                if (output.Interface == InputInterfaceType.DicomWeb)
+                {
+                    CheckDicomWebConnectionDetails("outputResources", errors, output.ConnectionDetails);
+                }
+            }
+
             details = string.Join(' ', errors);
             return errors.Count == 0;
         }
+
+        private static void CheckDicomWebConnectionDetails(string source, List<string> errors, DicomWebConnectionDetails connection)
+        {
+            if (connection.AuthType != ConnectionAuthType.None && string.IsNullOrWhiteSpace(connection.AuthId))
+            {
+                errors.Add($"One of the '{source}' has authType of '{connection.AuthType}' but does not include a valid value for 'authId'");
```

Comment:
`connection.AuthType` is an `enum`? if so, I recommend using `{connection.AuthType:F}` instead.